JASP-Engine/JASP/R/ancovabayesian.R

```diff
@@ -13,6 +13,17 @@ AncovaBayesian	 <- function (dataset = NULL, options, perform = "run", callback
 	if (is.null (base::options ()$BFfactorsMax))
 		base::options (BFfactorsMax = 5)
 
+	state <- .retrieveState ()
+	print(state)
```

Comment:
No, I am very keen on keeping it in.

On 08 Jun 2015, at 19:09, Jonathon Love <notifications@github.com<mailto:notifications@github.com>> wrote:


In JASP-Engine/JASP/R/ancovabayesian.R<https://github.com/jasp-stats/jasp-desktop/pull/690#discussion_r31936557>:

> @@ -13,6 +13,17 @@ AncovaBayesian      <- function (dataset = NULL, options, perform = "run", callback
>       if (is.null (base::options ()$BFfactorsMax))
>               base::options (BFfactorsMax = 5)
>
> +     state <- .retrieveState ()
> +     print(state)


hey, do you want to get rid of this print statement?

—
Reply to this email directly or view it on GitHub<https://github.com/jasp-stats/jasp-desktop/pull/690/files#r31936557>.